codec-http2/src/test/java/io/netty/handler/codec/http2/Http2FrameCodecTest.java

```diff
@@ -740,10 +767,21 @@ public void iterateActiveStreams() throws Exception {
         Http2FrameStream idleStream = frameCodec.newStream();
 
         final Set<Http2FrameStream> activeStreams = new HashSet<>();
-        frameCodec.forEachActiveStream(stream -> {
-            activeStreams.add(stream);
-            return true;
+        final AtomicReference<Http2Exception> errorRef = new AtomicReference<>();
+        channel.eventLoop().execute(() -> {
+            try {
+                frameCodec.forEachActiveStream(stream -> {
```

Comment:
we need to call this on the `EventLoop` as the codec asserts that we are on the event loop.